src/options.tsx

```diff
@@ -204,13 +205,16 @@ export const evaluateOption: Record<Option, Handler> = {
     return result.length ? (
       <>
         <Title>
-          找到 {result.length} 個相異項目。
+          {i18n.t("schemaCompareDifferent", { count: result.length })}
```

Comment:
This is tricky, we will need a functional component like
```js
function Translation({ message, ...options }) {
  const { t } = useTranslation();
  return t(message, options);
}
```
to get the messages sync with the language chosen (I am new to the i18next library and I haven’t think of how to type the component correctly yet, but I believe this should be what `react-i18next` provide. I wonder if there are alternative i18n libraries that provide such components.)